cime_config/testdefs/testlist_clm.xml

```diff
@@ -538,6 +538,24 @@
       <option name="wallclock">00:20:00</option>
     </options>
   </test>
+  <test name="ERP_D_P36x2_Ld3" grid="f10_f10_mg37" compset="I1850Clm51BgcCrop" testmods="clm/mimics">
+    <machines>
+      <machine name="cheyenne" compiler="gnu" category="aux_clm"/>
+    </machines>
+    <options>
+      <option name="wallclock">00:20:00</option>
+      <option name="comment"  >Test soil_decomp_method = 'MIMICSWieder2015'</option>
+    </options>
+  </test>
+  <test name="SMS_Ld5_Mmpi-serial" grid="1x1_brazil" compset="IHistClm50BgcQianRs" testmods="clm/mimics">
+    <machines>
+      <machine name="izumi" compiler="gnu" category="aux_clm"/>
+    </machines>
+    <options>
+      <option name="wallclock">00:20:00</option>
+      <option name="comment"  >Test soil_decomp_method = 'MIMICSWieder2015'</option>
+    </options>
+  </test>
```

Comment:
You need to add a test for the new newton_krylov testmod.

Reply:
@ekluzek do you mean a test for what's in this directory:
`cime_config/usermods_dirs/newton_krylov_spinup`?
...otherwise, I haven't created a newton_krylov testmod. Should I? I think I should wait for @wwieder and Keith Lindsay to make progress because I don't know if the N-K solution will work for sure. What do you think @wwieder ?

Reply:
I'm optimistic that the N-K will work, but it's vapor ware right now.

Reply:
I linked #1457 to this conversation to avoid forgetting...